components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -159,10 +159,10 @@ import { InvalidGitpodYMLError } from "./config-provider";
 import { ProjectsService } from "../projects/projects-service";
 import { LocalMessageBroker } from "../messaging/local-message-broker";
 import { IDEOptions } from "@gitpod/gitpod-protocol/lib/ide-protocol";
-import { PartialProject } from "@gitpod/gitpod-protocol/src/teams-projects-protocol";
+import { PartialProject } from "@gitpod/gitpod-protocol/lib/teams-projects-protocol";
 import { ClientMetadata, traceClientMetadata } from "../websocket/websocket-connection-manager";
```

Comment:
What change that these src ref no longer break the leeway build? :thinking:

Reply:
I have no idea, I just stumbled over one occasion and then went through the code and fixed all others. 🤷